lighty-examples/lighty-gnmi-community-restconf-app/README.md

```diff
@@ -0,0 +1,81 @@
+# lighty.io gNMI/RESTCONF application
+This application provides gNMI south-bound utilized with RESTCONF interface to manage gNMI devices on the network.
+Application works as standalone SDN controller. It is capable to connect to gNMI devices and expose connected devices
+over RESTCONF north-bound APIs. In this application will be started gNMI simulator as gNMI target and all operation will
+be performed on this device.
+
+Application lighty.io gNMI/RESTCONF is pre-prepared with [Openconfig YANG models](src/main/assembly/resources/yangs).
+These models are used by both gNMI application and gNMI device simulator. Device have already prepared state/config data
+configured by [this](src/main/assembly/resources/simulator/initialJsonData.json) json file.
+For communication with gNMI device is required to use TLS communication with certificates and authorized
```

Comment:
with gNMI device there is required to use...

Reply:
done